web/src/enterprise/campaigns/detail/CampaignUpdateDiff.tsx

```diff
@@ -55,6 +55,7 @@ export interface CampaignDiff {
      */
     unmodified: GQL.Changeset[]
     deleted: GQL.IExternalChangeset[]
+    hasNotUpdated: boolean
```

Comment:
The `hasNotUpdated` sounds to me like it describes the state of the update operation itself, which is slightly confusing.

We all know naming things is the worst, but... how about something like this?

```suggestion
    containsHidden: boolean
```